src/screens/AddSiteFormScreen.js

```diff
@@ -151,6 +175,14 @@ class AddSiteFormScreen extends Component {
         this.props.updatePriceOption({priceOption: newPriceOption})
     };
 
+    onUpdateCellProviderOption = (newCellProviderOption) => {
+        this.props.updateCellProviderOption({cellProviderOption: newCellProviderOption})
+    };
+
+    onUpdateCellStrengthOption = (newCellStrengthOption) => {
+        this.props.updateCellStrengthOption({cellStrengthOption: newCellStrengthOption})
+    };
+
```

Comment:
All of these things above for picker and text changing are the same pattern that we've always done.